core/src/main/java/com/linecorp/armeria/common/util/Sampler.java

```diff
@@ -42,45 +42,52 @@
  *
  * <p>Forked from brave-core 5.6.3 at 3be55b5cccf881104bdd80c93e97d2575b83952d
  */
-// abstract for factory-method support on Java language level 7
-public abstract class Sampler {
-
-    public static final Sampler ALWAYS_SAMPLE = new Sampler() {
-        @Override
-        public boolean isSampled() {
-            return true;
-        }
-
-        @Override public String toString() {
-            return "AlwaysSample";
-        }
-    };
+@FunctionalInterface
+public interface Sampler<T> {
+    /**
+     * Returns a sampler, given a rate expressed as a floating point number between {@code 0.0} and {@code 1.0}.
+     *
+     * @param rate minimum sample rate is 0.01, or 1% of traces
```

Comment:
- sample rate -> sampling rate
- If the minimum sampling rate is `0.01`, then we have to say `between {@code 0.01} and {@code 1.0}`, no?

Reply:
Thank you for review. Fixed in 844219f.
(Have been fixed a similar comment in `CountingSampler`.)